plugins/package-managers/bazel/src/main/kotlin/Bazel.kt

```diff
@@ -168,6 +169,24 @@ class Bazel(
             }
         }
 
+    override fun findPackageManagerDependencies(
+        analysisRoot: File,
+        managedFiles: Map<PackageManager, List<File>>,
+        analyzerConfig: AnalyzerConfiguration
+    ): PackageManagerDependencyResult {
+        // If both Bazel and Conan are enabled, they should not run in parallel, because this may cause conflicts with
+        // the Conan HOME files (see https://github.com/oss-review-toolkit/ort/pull/10357). Therefore, Conan is always
+        // run before Bazel.
```

Comment:
Nit: To better match the used `mustRunAfter` property, should this say "Bazel is always run after Conan"?